File: apps/websocket-skyblock/src/mongodb/collections/RollStats.cs
using System.Runtime.Serialization;
using MongoDB.Bson.Serialization.Attributes;

public class RollStats
{
    [BsonId]
    [DataMember]
    public MongoDB.Bson.ObjectId _id { get; set; }
    /// <summary>
    /// Number of losses during a /roll battle
    /// </summary>
    public int Loses = 0;
    /// <summary>
    /// User ID of the discord member.
    /// </summary>
    public ulong UserId;
    /// <summary>
    /// Number of wins during a /roll battle
    /// </summary>
    public int Wins = 0;

    public RollStats(ulong _UserId)
    {
        UserId = _UserId;
    }
}

File: apps/discord-bot/app/src/main/java/discordbot/mongodb/collections/RollStats.java
package discordbot.mongodb.collections;

import org.bson.codecs.pojo.annotations.BsonId;
import org.bson.types.ObjectId;

import kotlin.ULong;

public class RollStats {
    @BsonId
    public ObjectId _id;
    /**
     * The number of losses during a /roll battle.
     */
    public Integer Lose;
    /**
     * User ID of the discord member.
     */
    public ULong UserId;
    /**
     * The number of wins during a /roll battle.
     */
    public Integer Wins;

    public RollStats(ULong _UserId) {
        UserId = _UserId;
    }
}